threes.py

```diff
@@ -106,20 +111,20 @@ def play_game_interactive():
 
     while True:
         m, tileset, valid = game.send(move)
-        print to_val(m)
+        print(to_val(m))
 
         if not valid:
-            print "Game over."
-            print "Your score is", to_score(m).sum()
+            print("Game over.")
+            print("Your score is", to_score(m).sum())
             break
 
-        print 'next tile:', list(to_val(tileset))
+        print('next tile:', list(to_val(tileset)))
 
         movelist = ''.join('UDLR'[i] for i in valid)
         while True:
-            move = raw_input('Move [%s]? ' % movelist).upper()
+            move = input('Move [%s]? ' % movelist).upper()
```

Comment:
This breaks Py2 compatibility and introduces an eval vulnerability. Please use a compatibility function to avoid breaking Py2.